app/src/main/java/com/hover/stax/addChannels/AddChannelsFragment.kt

```diff
@@ -230,21 +239,6 @@ class AddChannelsFragment : Fragment(), ChannelsAdapter.SelectListener, CountryA
         AddChannelsFragmentDirections.actionNavigationLinkAccountToNavigationHome()
     }
 
-    //channels will be loaded only once after install then deferred to weekly.
-    private fun refreshChannelsIfRequired() {
```

Comment:
This was preventing channels from updating right away for a fresh install. Don't need our own boolean, work manager handles this for us.